packages/@aws-cdk/assert/lib/assertions/have-resource.ts

```diff
@@ -1,6 +1,11 @@
 import { Assertion, JestFriendlyAssertion } from "../assertion";
 import { StackInspector } from "../inspector";
 
+/**
+ * Magic value to signify that a certain key should be absent from the property bag
+ */
+export const ABSENT = '{{ABSENT}}';
```

Comment:
![Noice!](https://media1.tenor.com/images/7c02ebe55ca7a950f816d2609f37086f/tenor.gif?itemid=11928987)

Just wondering if this should be a `Symbol` to avoid possibly conflicting with a legitimate string value?

Reply:
I considered that for a moment, and I guess worrying about jsii compatibility is a little premature at the moment, but I didn't do that because it wouldn't be representable there.